modules/tinymce/src/themes/silver/main/ts/ButtonState.ts

```diff
@@ -7,39 +7,64 @@ import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
 
 import { ReadyUiReferences } from './modes/UiReferences';
 
+export interface SetEnabledEvent {
+  readonly eventType: 'setEnabled';
+  readonly shouldDisable: boolean;
```

Comment:
Seems a little backwards to have a event called `SetEnabledEvent` then have the value be `shouldDisable` maybe better to call that `enabled` or just `value` and flip the logic or rename the event to`setDisabled` but I guess disabled is a bad word?